app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java

```diff
@@ -102,6 +102,13 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public void onBackPressed() {
```

Comment:
pay attention that we are going to use the same method for closing the panel when it is open